ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/local-users.j2

```diff
@@ -1,23 +1,23 @@
 {# eos - local users #}
 {% if local_users is arista.avd.defined %}
 !
-{%     for local_user in local_users | arista.avd.natural_sort %}
+{%     for local_user in local_users | arista.avd.convert_dicts('name') | arista.avd.natural_sort('name') %}
 {%         set local_user_cli = "username " ~ local_user %}
```

Comment:
Remember to change the base `variable` to `variable.key`. In this example it should be `local_user.name`.
Similar across all files in both PRs.